docs/samples/basic_config/aggregation_job_driver.yaml

```diff
@@ -29,3 +29,8 @@ maximum_attempts_before_failure: 10
 # Number of sharded database records per batch aggregation. Must not be greater
 # than the equivalent setting in the collection job driver. (required)
 batch_aggregation_shard_count: 32
+
+# Defines the number of shards to break report & aggregation metric counters into. Increasing this
+# value will reduce the amount of database contention during report uploads & aggregations, while
+# increasing the cost of getting task metrics.
+task_counter_shard_count: 32
```

Comment:
I think we could leave this out of the basic file, since it has a default value. (we currently do the same with the upload counter shard count)